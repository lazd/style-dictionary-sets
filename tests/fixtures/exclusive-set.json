{
  "accent-content-color-hover": {
    "value": "{accent-color-1000}"
  },
  "accent-color-1000": {
    "sets": {
      "spectrum": {
        "value": "rgb(0, 84, 182)"
      },
      "express": {
        "value": "rgb(64, 70, 202)"
      }
    }
  }
}
