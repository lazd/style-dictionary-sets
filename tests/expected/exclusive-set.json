{
  "accent-content-color-hover": {
    "ref": "{accent-color-1000}",
    "value": "rgb(64, 70, 202)"
  },
  "accent-color-1000": {
    "value": "rgb(64, 70, 202)"
  }
}
